var/spack/repos/builtin/packages/storm/package.py

```diff
@@ -13,16 +13,18 @@ class Storm(Package):
     Storm provides a set of general primitives for doing realtime computation.
     """
 
-    homepage   = "https://www-eu.apache.org/dist/storm"
-    url        = "https://www-eu.apache.org/dist/storm/apache-storm-2.1.0/apache-storm-2.1.0.tar.gz"
-    url_list   = homepage
+    homepage = "https://storm.apache.org/"
+    url      = "https://archive.apache.org/dist/storm/apache-storm-2.2.0/apache-storm-2.2.0.tar.gz"
+    url_list = "https://archive.apache.org/dist/storm/"
```

Comment:
```suggestion
    list_url = "https://archive.apache.org/dist/storm/"
```
I wonder how long this typo has been in there...

Reply:
Sorry, fixed in https://github.com/spack/spack/pull/17523/commits/0f6f4c838f41240ebeac08b43c98f6f968ec3939
I changed the `list_url` and `list_depth` value.